src/org/camunda/latera/bss/connectors/hid/hydra/Account.groovy

```diff
@@ -203,27 +376,50 @@ trait Account {
   """, true)
   }
 
+  /**
+   * Get account total balance
+   * @param accountId     {@link java.math.BigInteger BigInteger}
+   * @param operationDate {@link java.time.Temporal Any date type}. Optional. Default: current datetime
+   * @return Double with account total sum
+   */
   Double getAccountBalanceTotal(
     def accountId,
     Temporal operationDate = local()
   ) {
-    return toFloatSafe(getAccountBalance(accountId, operationDate)?.n_sum_total).doubleValue()
+    return toFloatSafe(getAccountBalance(accountId, operationDate)?.n_sum_total)?.doubleValue()
   }
 
+  /**
+   * Get account free balance
+   * @param accountId     {@link java.math.BigInteger BigInteger}
+   * @param operationDate {@link java.time.Temporal Any date type}. Optional. Default: current datetime
+   * @return Double with account free sum
+   */
   Double getAccountFree(
     def accountId,
     Temporal operationDate = local()
   ) {
-    return toFloatSafe(getAccountBalance(accountId, operationDate)?.n_sum_free).doubleValue()
+    return toFloatSafe(getAccountBalance(accountId, operationDate)?.n_sum_free)?.doubleValue()
   }
 
+  /**
+   * Get account actual charge logs total
+   * @param accountId {@link java.math.BigInteger BigInteger}
+   * @return Double with total sum of account charge logs
```

Comment:
I prefer not to do that.

If certain types on arguments, default values and return types are declared in javadoc, they will be shown in generated documentation.

But in case of:
1. Groovy's named arguments
2. `def` keyword which is used to describe, for example, ids which can be either String or Integer, or BigInteger
such documentation will not help you to understand what should be passed into method call.

That's why I've added possible types for such arguments to javadoc. But using it everywhere will be really just a waste of time.